src/lib.rs

```diff
@@ -301,7 +307,7 @@ where
     T: crate::TimeSource,
 {
     raw_volume: RawVolume,
-    volume_mgr: &'a VolumeManager<D, T, MAX_DIRS, MAX_FILES, MAX_VOLUMES>,
+    volume_mgr: &'a mut VolumeManager<D, T, MAX_DIRS, MAX_FILES, MAX_VOLUMES>,
```

Comment:
I think this breaks the ability to open multiple volumes/directories/files at a time, which I added recently?

Reply:
Oh, didn't know that.. I guess the borrow checker would not like that